.ci/scripts/run-gradle.sh

```diff
@@ -31,5 +31,10 @@ if ! uname -a | grep -q MING; then
   export GLIBC_VERSION=$(ldd --version | grep '^ldd' | sed 's/.* \([1-9]\.[0-9]*\).*/\1/')
 fi
 
+# Running on 2-core machines without ramdisk can make this value be 0
+if [[ "$MAX_WORKERS" == "0" ]]; then
+  MAX_WORKERS=1
```

Comment:
Should we just make it the number of CPUs then in that case?

Reply:
Well, I think it only ends up as 0 if the number of CPUs is 1 or 2... I was thinking that on a 2-CPU machine, there might not really be enough resources to effectively run two tasks in parallel, what do you think?

Reply:
We can leave it at one and tweak later.